flow-nodes/api-builder-plugin-logger/test/test.js

```diff
@@ -72,60 +86,44 @@ describe('flow-node logger', () => {
 		// validation to avoid potential issues when API Builder loads your
 		// node.
 		it('should define valid flow-nodes', () => {
-			expect(runtime.validate()).to.not.throw;
+			// if this is invalid, it will throw and fail
+			plugin.validate();
 		});
 	});
 
 	describe('#hello', () => {
 		it('should error when logging at unknown level', async () => {
-			// Invoke #hello with a non-number and check error.
-			const flowNode = runtime.getFlowNode('logger');
-
-			const result = await flowNode.log({
+			const { value, output } = await flowNode.log({
 				level: 'foobar'
 			});
 
-			expect(result.callCount).to.equal(1);
-			expect(result.output).to.equal('error');
+			expect(output).to.equal('error');
 
-			expect(result.args[0]).to.equal(null);
-			expect(result.args[1]).to.be.instanceOf(Error)
-				.and.to.have.property('message', 'invalid log level: foobar');
-			expect(result.context).to.be.an('Object');
-			expect(result.context.error).instanceOf(Error)
+			expect(value).to.be.instanceOf(Error)
 				.and.to.have.property('message', 'invalid log level: foobar');
 		});
 
 		it('should log with default log level', async () => {
-			simple.mock(console, 'trace', () => {});
-			const flowNode = runtime.getFlowNode('logger');
-
-			const result = await flowNode.log({ message: 'Hello' });
-
-			expect(console.trace.callCount).to.equal(1);
-			expect(console.trace.lastCall.arg).to.equal('Hello');
-			expect(result.callCount).to.equal(1);
-			expect(result.output).to.equal('next');
-			expect(result.args).to.deep.equal([]);
-			expect(result.context).to.be.undefined;
+			simple.mock(logger, 'trace', () => {});	
+			const { value, output } = await flowNode.log({
+				message: 'Hello'
+			});
+			
+			expect(output).to.equal('next');
+			expect(value).to.be.undefined;
 		});
 
 		for (const level of [ 'trace', 'debug', 'warn', 'error' ]) {
 			it(`should log at log level ${level}`, async () => {
-				simple.mock(console, level, () => {});
-				const flowNode = runtime.getFlowNode('logger');
-
-				const result = await flowNode.log({
+				 
+				simple.mock(logger, level, () => {});	
+				const { value, output } = await flowNode.log({
 					level,
 					message: 'Hello'
 				});
-
-				expect(console[level].callCount).to.equal(1);
-				expect(console[level].lastCall.arg).to.equal('Hello');
-				expect(result.callCount).to.equal(1);
-				expect(result.output).to.equal('next');
-				expect(result.args).to.deep.equal([]);
-				expect(result.context).to.be.undefined;
+				expect(logger[level].lastCall.arg).to.equal('Hello');
```

Comment:
You accidentally deleted the `callCount` check.  Otherwise, it might be possible for every log message to be logged at 'trace' and we'd never know it.

Reply:
The team has instructed to remove this check and this call is not needed anymore. Do we need to add this back in all unit tests?

Reply:
This is added now.